Jenkinsfile

```
@@ -40,4 +40,4 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipelineDAOS(['distros': ['centos7', 'centos8', 'leap15']])
+packageBuildingPipelineDAOS(['distros': ['el7', 'el8', 'leap15']])
```

We are not aliasing centos7 into el7 at this point.

```suggestion
packageBuildingPipelineDAOS(['distros': ['centos7', 'el8', 'leap15']])

```

---

I tried building with `centos7` and the `createrepo` step never ran, so I figured that maybe there was some broken magic in the pipeline goop. As the primary thrust of this was to get leap15 builds working again I figured that I'd revisit this today.